src/libs/notifications.js

```diff
@@ -147,8 +147,9 @@ const showNotification = ({ id, timeout }) => {
     ]),
     container: 'top-right',
     dismiss: { duration: !!timeout ? timeout : 0, click: false, touch: false },
-    animationIn: ['animate__animated', 'animate__slideInRight'],
-    animationOut: ['animate__animated', 'animate__slideOutRight'],
+    animationIn: ['animate__animated', 'animate__fadeIn'],
+    animationOut: ['animate__animated', 'animate__fadeOut'],
```

Comment:
I might leave the animationOut as a slide? Just since that one (I think) needs to be initiated by the user, a harder out animation might be nice for stronger immediate feedback, but it would disrupt the symmetry there. I would defer to UX.